providers/nfs_provider/deploy/nfs-provider-deployment.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 The SODA Authors.
```

Comment:
I think this deploy yaml can have meta service part as well , otherwise nfs deployment alone will not be useful

Reply:
Done, metaservice is added to the deployment